SIPs/sip-1.md

```diff
@@ -0,0 +1,85 @@
+---
+sip: 1
+title: SIP Process and Guidelines
+author: Team TSS
+discussions-to: N/A
+status: Draft
+category: Informational
+created: 2018-Jun-4
```

Comment:
Shouldn't it be the current date?

Reply:
Good catch, I forgot to change that year 👍🏼